src/ridgeplot/_color/colorscale.py

```diff
@@ -54,7 +55,19 @@ def validate_and_coerce_colorscale(
 def list_all_colorscale_names() -> list[str]:
     """Get a list of all available continuous colorscale names.
 
-    .. versionadded:: 0.1.21
-        Replaced the deprecated function ``get_all_colorscale_names()``.
+    .. deprecated:: 0.1.31
+       This function is deprecated and will be removed in a future version.
+       Please use ``px.colors.named_colorscales()`` from Plotly Express for
```

Comment:
**Small tip:** I changed `` `px.colors.named_colorscales()` `` to `` :func:`px.colors.named_colorscales() <plotly.express.colors.named_colorscales>` ``, which allows Sphinx to automatically link to Plotly's API reference.

Currently this is how it looked before:

![image](https://github.com/user-attachments/assets/b505cc78-d91f-43d2-87a9-9700c62d1886)

and now:

![image](https://github.com/user-attachments/assets/e9d4dc91-dd53-40eb-9118-97da0434439b)


The docs are built automatically for every PR. So you can check this here: https://ridgeplot--262.org.readthedocs.build/en/262/api/public/ridgeplot.list_all_colorscale_names.html#ridgeplot.list_all_colorscale_names

To build the docs and check how it would look like locally first, you could run `tox -e docs-static`

Reply:
Great tip. Thanks :)

Reply:
Hey, so we can generate a doc thorugh the Docstring?

Reply:
Yeh it's pretty neat but also pretty standard in the Python ecosystem. We use Sphinx to build the docs and use the [autodoc](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html) extension to automatically generate the API reference from docstrings